float64[3] left_xyz
float64[3] right_xyz
---
bool success